File: Aplicaciones Distribuidas/PARCIAL2/HILOS_DOTNET_GRUPO05/TALLER_HILOS_PRACTICA3/APLICATIVO/176_TALLER_HILOS_GRUPO05/Vista/BancoVista.cs
using _176_TALLER_HILOS_GRUPO05.Modelo;
using System;
using System.CodeDom;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;

namespace _176_TALLER_HILOS_GRUPO05.Vista
{
    public class BancoVista
    {
        public void imprimirErrorTransferencia(BancoModelo banco, int cuentaOrigen, double cantidad)
        {
            Console.ForegroundColor = ConsoleColor.Red;
            Console.WriteLine("Fondos Insuficientes!!!\n");
            Console.ForegroundColor = ConsoleColor.Blue;
            Console.WriteLine("\tCuenta: " + cuentaOrigen.ToString("0.##"));
            Console.WriteLine("\tSaldo: " + banco.cuentas[cuentaOrigen].ToString("0.##"));
            Console.WriteLine("\tCantidad: " + cantidad.ToString("0.##"));
        }

        public void imprimirTranserenciaExito(int cuentaOrigen)
        {
            Console.ForegroundColor = ConsoleColor.Green;
            Console.WriteLine("Transferencia realizada\n");
            Console.WriteLine("\tCuenta: " + cuentaOrigen);
        }

        public void imprimirDetallesTransferencia(
            BancoModelo banco,
            int cuentaOrigen,
            int cuentaDestino,
            double cantidad,
            double saldoTotal
        )
        {
            Console.WriteLine(Thread.CurrentThread.ManagedThreadId + "\n");
            Console.WriteLine(cantidad.ToString("0.##") + " de " + cuentaOrigen + " para " + cuentaDestino);
        }
    }
}

File: Aplicaciones Distribuidas/PARCIAL1/05. HILOS_NETBEANS_CHINLLE_CHUQUIMARCA_COBACANGO/TALLER_HILOS_PRACTICA3/Aplicativo/178_TALLER_HILOS_GRUPO05/src/ec/edu/monster/vista/BancoVista.java
package ec.edu.monster.vista;

import ec.edu.monster.modelo.BancoModelo;

public class BancoVista {
    public static final String ANSI_RESET = "\u001B[0m";
    public static final String ANSI_BLACK = "\u001B[30m";
    public static final String ANSI_RED = "\u001B[31m";
    public static final String ANSI_GREEN = "\u001B[32m";
    public static final String ANSI_YELLOW = "\u001B[33m";
    public static final String ANSI_BLUE = "\u001B[34m";
    public static final String ANSI_PURPLE = "\u001B[35m";
    public static final String ANSI_CYAN = "\u001B[36m";
    public static final String ANSI_WHITE = "\u001B[37m";

    public void imprimirErrorTransferencia(BancoModelo banco, int cuentaOrigen, double cantidad) {
        System.out.println(ANSI_RED + "Fondos Insuficientes!!!\n" + ANSI_RESET);
        System.out.println(ANSI_BLUE + "\tCuenta: " + ANSI_RESET + cuentaOrigen);
        System.out.println(ANSI_BLUE + "\tSaldo: " + ANSI_RESET + banco.cuentas[cuentaOrigen]);
        System.out.println(ANSI_BLUE + "\tCantidad: " + ANSI_RESET + cantidad);
    }

    public void imprimirTranserenciaExito(int cuentaOrigen) {
        System.out.println(ANSI_GREEN + "Transferencia realizada\n" + ANSI_RESET);
        System.out.println(ANSI_BLUE + "\tCuenta: " + ANSI_RESET + cuentaOrigen);
    }

    public void imprimirDetallesTransferencia(
            BancoModelo banco,
            int cuentaOrigen,
            int cuentaDestino,
            double cantidad,
            double saldoTotal
    ) {
        System.out.println(Thread.currentThread() + "\n");
        System.out.printf("%10.2f de %d para %d\n", cantidad, cuentaOrigen, cuentaDestino);
        System.out.printf(ANSI_BLUE + "Saldo Total:" + ANSI_RESET + "%10.2f\n", saldoTotal);
    }
}